Sources/Clickstream/Dependency/Clickstream/AutoRecord/AutoRecordEventClient.swift

```diff
@@ -64,12 +64,11 @@ class AutoRecordEventClient {
         if lastEngageTime > 0 {
             event.addAttribute(lastEngageTime, forKey: Event.ReservedAttribute.ENGAGEMENT_TIMESTAMP)
         }
-        recordEvent(event)
-
-        isEntrances = false
         lastScreenName = screenName
         lastScreenPath = screenPath
         lastScreenUniqueId = screenUniqueId
+        recordEvent(event)
```

Comment:
add test case for this change

Reply:
this logic is covered by this [test case](https://github.com/awslabs/clickstream-swift/blob/main/Tests/ClickstreamTests/Clickstream/AutoRecordEventClientTest.swift#L91-L98)

Reply:
Why the test case was passed without this fix? Is it an intermitent issue?

Reply:
Yes, the probability of this problem occurring is extremely low, and it is difficult to detect through unit testing. Only by conducting thousands of app startup tests on a real machine can it be discovered. So this issue only found by device farm integration test.
This modification ensures that the screen view name is always included.